mesheryctl/internal/cli/root/system/reset_test.go

```diff
@@ -0,0 +1,85 @@
+package system
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	log "github.com/sirupsen/logrus"
+)
+
+//var update1 = flag.Bool("update", true, "update golden files")
+
+// This is an Integration test
+func TestResetCmdIntegration(t *testing.T) {
+	// skipping this integration test with --short flag
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
```

Comment:
@sayantan1413 This test isn't an integration test.

Reply:
Yeah @piyushsingariya I have removed it.